manifests/init.pp

```diff
@@ -13,13 +13,13 @@
 # @param gpgkeyfingerprint String with the ID from the gpg key that signed the packages
 # @param ensure Install or deinstall all related components
 class lldpd (
-  Boolean $manage_facts,
-  Boolean $manage_service,
   Boolean $manage_repo,
```

Comment:
I think these two parameters should be set explicitly to true, for better readability.

Reply:
As @ekohl mentioned above, we removed the default here because it's not the same on every operating system and it might confuse people because reference.md/init.pp says default is X but it's actually Z.

Reply:
But according to REFERENCE.md the parameter ```manage_repo``` is set to true by default.

Reply:
ups, forgot to update it. regenerated REFERENCE.md